agents/template/src/main/scala/org/broadinstitute/transporter/transfer/TransferRunner.scala

```diff
@@ -12,7 +14,7 @@ trait TransferRunner[In, Progress, Out] {
     * Initialize the transfer described by the given request, and
     * emit enough information to push the first chunk of data.
     */
-  def initialize(request: In): Either[Progress, Out]
+  def initialize(request: In): TransferState[In, Progress, Out]
```

Comment:
Vice versa

Reply:
+1 for consistency, would use the single-letter variants everywhere now